sox.dialog.js

```diff
@@ -257,11 +240,7 @@
           const searchQuery = $(this).val();
           $('.sox-new-version-details').hide();
           $('#sox-settings-dialog .sox-feature').each(function() {
-            if ($(this).find('label').text().toLowerCase().indexOf(searchQuery) == -1) {
-              $(this).hide();
-            } else {
-              $(this).show();
-            }
+            $(this).find('label').text().match(new RegExp(searchQuery, 'i')) ? $(this).show() : $(this).hide();
```

Comment:
I don't think we need to use a regex here -- the `.toLowerCase().indexOf(...)` works just fine (and [is faster](https://jsperf.com/case-insensitive-indexof))

Reply:
Actually depends on the browser. Anyway, reverted the commit.